server/app.py

```diff
@@ -1,10 +1,21 @@
 from flask import Flask
 from api.ping_handler import ping_handler
 from api.home_handler import home_handler
+from api.auth_handler import create_auth_blueprint
+from api.after_auth_handler import create_after_Auth_blueprint
+from flask_cors import CORS
+from config import basic_auth_configSetup
 
 app = Flask(__name__)
+app.secret_key = "Simerpreet"
```

Comment:
What is the secret key used for? Is it something that could be in the .env/.example.env instead?

Reply:
oh! sorry. Secret key is for session object. It should be random so no one can guess. I made changes. please check latest version.